src/domain/cohort.js

```diff
@@ -5,6 +5,7 @@ import dbClient from '../utils/dbClient.js'
  * @returns {Cohort}
  */
 export async function createCohort() {
+
```

Comment:
It looks like this is a linting error. You'll need to fix this before you can merge.